.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: "Build & Test"
+on:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: password
+        ports:
+          - 25838:3306
+
+    env:
+      DATABASE_HOST: "127.0.0.1"
+      DATABASE_USER: root
+      DATABASE_PASSWORD: password
+      DATABASE_PORT: 25838
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: "install Nix"
+      uses: cachix/install-nix-action@v11
+
+    - name: "Cache Nix packages"
+      uses: cachix/cachix-action@v6
+      with:
+        name: svsticky-constipated-koala
+        signingKey: ${{ secrets.CACHIX_SIGNING_KEY }}
+    
+    - name: "Cache Yarn packages"
+      uses: actions/cache@v2
+      with:
+        path: node_modules
+        key: yarn-${{ hashFiles('yarn.lock') }}
+        restore-keys: |
+          yarn-
+
+    - name: "Execute Ruby linter"
+      run: nix-shell --run "bundle exec rubocop"
+
+    - name: "Execute Javascript linter"
+      run: nix-shell --run "npx prettier -c app/assets/javascripts/*"
+    
```

Comment:
It is probably a good idea to add a CI step that checks whether the Yarn lockfile is up to date here. I believe this is `yarn check --frozen-lockfile` but not sure.